client/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "malefashion-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@paypal/paypal-js": "^5.1.4",
+    "@paypal/react-paypal-js": "^7.8.1",
+    "@reduxjs/toolkit": "^1.9.0",
+    "@stripe/react-stripe-js": "^1.15.0",
```

Comment:
Read this article, it will explain the packages which should be placed in `devDependencies` or `dependencies`

https://sebhastian.com/npm-save-dev/